client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -11,6 +11,7 @@ import { connect } from 'react-redux';
 import { getCurrentlyEditingProduct } from '../../state/ui/products/selectors';
 import { editProduct, editProductAttribute } from '../../state/ui/products/actions';
 import ProductForm from './product-form';
+import ProductHeader from './product-header';
```

Comment:
Since the save button and breadcrumbs will be on more than just product pages (i.e. add order, settings, add coupon, etc), let's not make this a product specific component. How about a more generic name and location?